win32ss/drivers/videoprt/dispatch.c

```diff
@@ -1029,7 +1029,11 @@ IntVideoPortDispatchFdoPnp(
             if (NT_SUCCESS(Status) && NT_SUCCESS(Irp->IoStatus.Status))
                 Status = IntVideoPortFilterResourceRequirements(DeviceObject, Irp);
             Irp->IoStatus.Status = Status;
-            Irp->IoStatus.Information = 0;
+            if (!NT_SUCCESS(Status) && Irp->IoStatus.Information != 0)
```

Comment:
This shouldn't be done here, but in whatever function has allocated it.

Reply:
Moved.